roboteam_ai/src/skills/Attack.cpp

```diff
@@ -40,30 +39,6 @@ bt::Node::Status Attack::onUpdate() {
     command.kicker_vel = shotData.kickSpeed;
     command.geneva_state = shotData.genevaState;
 
-
```

Comment:
This needs to be re - implemented somehow. (maybe just with 'canmoveInDefenseArea' etc from GTP